content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -148,6 +148,14 @@ means that if this pod is running and a matching taint is added to the node, the
 the pod will stay bound to the node for 3600 seconds, and then be evicted. If the
 taint is removed before that time, the pod will not be evicted.
 
+{{< note >}}
+The evicted pod with `tolerationSeconds` field has the possibility to be scheduled to the prior node again. To avoid such behavior:
```

Comment:
I think we are lacking some subsections for the whole "concepts" section. @sftim, could you advice?

In particular, I feel like there should be subtitle before line 132.


Other than that, we have to stress the fact that, even with "tolerationSeconds", this is still a toleration, and for that reason the Pod can get scheduled in the same node again.

Instead of "to avoid such behavior", I would say "If this is not what you want", then "you can add a `NoSchedule` taint additionally to the `NoExecute` taint".

Then, I don't think you have to distinguish system-level taints (what is that anyways?). Only the second point is relevant.

Reply:
Sure, feel free to add a heading before line 132 - no objection from me for doing that.

Reply:
The problem is that then that heading seems isolated. We don't have a heading for "taints" or "tolerations". But maybe we can fix that in a follow up.

Reply:
I  think system-level taints is like `node.kubernetes.io/unreachable` when node is down. And controller will add both `NoExcute` and `NoSchedule` taint to the node.

Reply:
That's already covered in https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/#taint-nodes-by-condition

The fact that they are added automatically doesn't change how you write tolerations for them :)

Reply:
BTW, PR #25998 is improving some of the documentation in this area.